backend/src/main/java/com/bakdata/conquery/apiv1/MeProcessor.java

```diff
@@ -34,31 +34,11 @@ public FEMeInformation getUserInformation(@NonNull User user){
 		return FEMeInformation.builder()
 			.userName(user.getLabel())
 			.hideLogoutButton(!user.isDisplayLogout())
-			.groups(FEGroup.from(AuthorizationHelper.getGroupsOf(user, storage)))
+			.groups(AuthorizationHelper.getGroupsOf(user, storage).stream().map(g -> new IdLabel<GroupId>(g.getId(),g.getLabel())).collect(Collectors.toList()))
```

Comment:
How about making this more readable? ;-)

Reply:
Can do :)

Reply:
ditto